components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/Fields.java

```diff
@@ -91,4 +93,11 @@ private static String formatDuration(Duration duration) {
 
         return s.toString().trim();
     }
+
+    private static String formatBigDecimal(BigDecimal value) {
```

Comment:
Maybe we should rename this to "formatSerializationFactor" since we won't want to format all BigDecimal's this way (returning empty string if the value is zero).